exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -66,9 +74,44 @@ impl JavaServiceRuntime {
         }
     }
 
-    /// Returns internal service proxy.
-    pub fn service_proxy(&self) -> ServiceProxy {
-        self.service_proxy.clone()
+    /// Loads an artifact from the specified location involving verification of the artifact.
+    /// Returns an unique service artifact identifier that must be specified in subsequent
+    /// operations with it.
+    pub fn load_artifact(&self, artifact_uri: &str) -> String {
+        unwrap_jni(self.executor.with_attached(|env| {
+            let artifact_uri: JObject = unwrap_jni(env.new_string(artifact_uri)).into();
```

Comment:
`unwrap_jni` shouldn't be needed here, as you can use question mark operator

Reply:
Also, why load_artifact, an operation that can fail for numerous reasons, performs an `unwrap`?